docs/examples/driver_examples/Qcodes_example_with_Keysight_B2200_femto_leakage_switch_matrix.ipynb

```diff
@@ -0,0 +1,1262 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import qcodes as qc\n",
+    "\n",
+    "from qcodes.instrument_drivers.Keysight.keysight_b220x import KeysightB220X\n",
+    "from pyvisa.errors import VisaIOError\n",
+    "\n",
+    "from IPython.display import display, Markdown"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "using physical instrument\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "c:\\users\\administrator\\pycharmprojects\\venvs\\qcodes_runtime\\lib\\site-packages\\pyvisa\\resources\\messagebased.py:569: UserWarning: read string doesn't end with termination characters\n",
+      "  return self.read()\n"
```

Comment:
Do you have any idea why this is printed? Would be nice to fix this warning

Reply:
I have to dig a bit more. I have seen it a couple of times after a reset, but its hard to reproduce.

Reply:
Looks like I used the wrong term char (`\r` instead of `\n`). I will change it.